azure-devops/azext_devops/dev/team/_help.py

```diff
@@ -28,6 +28,16 @@ def load_team_help():
     short-summary: Manage service endpoints/service connections
     """
 
+    helps['devops service-endpoint azurerm'] = """
+    type: group
+    short-summary: Manage Azure RM service endpoints/service connections
+    """
+
+    helps['devops service-endpoint github'] = """
```

Comment:
yes, but we do anticipate to author "update" command for every "create" command at some point of time

Reply:
I will let @geverghe decide.

Reply:
Proposal 1: az devops service-endpoint github create
Pros
a. All though only one command in group, there is an update API and leaves room for the update command if and when we decide to build it
Cons
b. One level deeper and not adhering to the guidance pertaining to if one command don't have a group
 
Proposal 2: az devops service-endpoint create-github-connection
Pros
a. Adherence to guidance and one level higher discover
Cons:
a. Need to deprecate this command when creating group
b. Maintenance cost
Would need to provide the command with deprecation warning for potentially one version (say 10)/reasonable amount of time, and hide the command on the release post that (11) and then wait till traffic dies down/reasonable amount of time before removing the command completely. During the time, in case there are any changes, there is maintenance cost as both - deprecated and new command, would need to be updated
 
Due to the extra costs associated with proposal 2 for limited gains - hypothesis at this step being the extra level of command depth does not affect user experience significantly, let us go with Proposal 1 and leave space for update. Ideally we should do update and thereby eliminate the conundrum and make service connection commands complete.